mmdet/models/detectors/panoptic_two_stage_segmentor.py

```diff
@@ -201,3 +204,74 @@ def simple_test(self, img, img_metas, proposals=None, rescale=False):
             result = dict(pan_results=pan_results)
             results.append(result)
         return results
+
+    def show_result(self,
```

Comment:
Make sure this API has the same arguments with the same order as that in basedetector.

Reply:
They are the same.